DSESwift/Utils/ValueParser.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ValueParser.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 30/10/15.
+//
+//
+
+import Foundation
+
+internal protocol BaseValueParser {
+    func getValueInOriginalFormat(dataPoint: DataPoint) -> AnyObject
+}
+
+internal class IntValueParser: BaseValueParser {
```

Comment:
shouldnt these just be functions? why wrap them in a class and protocol structure in the first place?